web/src/cypress/integration/favorites.ts

```diff
@@ -0,0 +1,107 @@
+import { testScreen } from '../support'
+import { Chance } from 'chance'
+const c = new Chance()
+
+testScreen('Favorites', testFavorites)
+
+function testFavorites(screen: ScreenFormat) {
+  describe('Rotation Favorites', () => {
+    let rot: Rotation
+    beforeEach(() => {
+      cy.createRotation()
+        .then(r => {
+          rot = r
+        })
+        .visit('/rotations')
+    })
+
+    it('should have favored rotations move to first on rotation list', () => {
+      cy.pageSearch(rot.name)
+      cy.get('#app')
+        .contains(rot.name)
+        .click()
+      cy.get('button[aria-label="Set as a Favorite Rotation"]')
+        .click()
+        .visit('/rotations')
+      cy.get('#app').contains(rot.name)
```

Comment:
In order to make sure it's first in the list, we need to assert its position within the list. I believe this can be achieved with `eq`.

e.g.: `cy.get('SomeListReference').eq(0)`

Reply:
From the tests running there could be anywhere from 1- 15 rotations in the rotation list, so I don't think testing for the position in the list would work well. I am not sure what would be the best practice to test for something like this.

Reply:
If you ensure that only one rotation is a favorite (going from 0 favorites -> 1 favorite), we can guarantee that asserting based off the first rotation in the list will be successful